ghub.org

```diff
@@ -57,7 +57,7 @@ this:
 #+END_SRC
 
 To be able to do that, Ghub needs to know who you want to talk to, who
-you are, and how you are going to proof the latter to the former.
+you are, and how you are going to prove the latter to the former.
```

Comment:
While proof is a verb, it's generally related to [baking](https://en.wikipedia.org/wiki/Proofing_%28baking_technique%29).